packages/expo-modules-autolinking/scripts/ios/react_import_patcher.rb

```diff
@@ -0,0 +1,72 @@
+# Copyright 2018-present 650 Industries. All rights reserved.
+
+module Expo
+  class ReactImportPatcher
+
+    public def initialize(installer, options)
+      @root = installer.sandbox.root
+      @module_dirs = get_module_dirs(installer)
+      @options = options
+    end
+
+    public def run!
+      args = [
+        'node',
+        '--eval',
+        'require(\'expo-modules-autolinking\')(process.argv.slice(1))',
```

Comment:
This is another place where we construct an array with `node --eval` command params. I think, next time we should move this to a separate util class 🤔

Reply:
maybe when there's another use case. i am hoping we can deprecate this file some days later e.g. in SDK 46. then we just need to remove the file entirely. 😁